examples/test/test.tf

```diff
@@ -0,0 +1,33 @@
+terraform {
+  required_providers {
+    tanzu-mission-control = {
+      source = "vmware/dev/tanzu-mission-control"
+    }
+  }
+}
+provider "tanzu-mission-control" {
+  endpoint            = "playground.tmc.cloud.vmware.com"                                  # optionally use TMC_ENDPOINT env var
+  vmw_cloud_api_token = "us9YyQZA5h5wUzxROle6f8gALI_LjsNlkSWxhN1TzF-tVlVeNrxeBRFhdAHvJCXL" # optionally use VMW_CLOUD_API_TOKEN env var
```

Comment:
please revoke ur token as soon as possible, this is a security concern. This is put in public